test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -0,0 +1,703 @@
+            	// Set values to N/A if they exceed the filterValue
+                if (parseInt(this.state.consolidatedData[i].platforms[platform].compare) > filterValue) {
+                    newArray[i].platforms[platform].compare = 'N/A';
+                }
+            }
+        }
+
+        this.setState({consolidatedData: newArray});
+        }
+    }
+
+    benchmarkFilter(value) {
+        let newArray = [];
+
+        //Always call color filter after applying benchmark filter, if no filter selected return original data
+        if (value.length === 0) {
+            this.setState({
+                consolidatedData: this.state.originalData
+            }, () => {
+            	// Indicate that colorFilter is being called second
+                this.colorFilter(false);
+            });
+        } else {
+            this.state.originalData.forEach(function (element) {
+                let found = false;
+
+                for (let i = 0; i < value.length; i++) {
+                    if (element.benchmarkName.indexOf(value[i]) !== -1) {
+                        found = true;
+                    }
+                }
+
+                if (found) {newArray.push(element);}
+            }.bind(this));
+
+            this.setState({
+                consolidatedData: newArray
+            }, () => {
+                this.colorFilter(false);
+            });
+        }
+    }
+    // Enter values in the proper fields before saving to the test/baseline array
+    handleEntry(element, metric, type) {
+        if (type === 'test') {
+            return {testScore: element.aggregateInfo[0].metrics[metric].value.mean, 
+                testJdkDate: element.aggregateInfo[0].benchmarkProduct.split("-")[1],
+                testJdk: element.aggregateInfo[0].benchmarkProduct,
+                testCI: element.aggregateInfo[0].metrics[metric].value.CI,
+                testSdkResource: element.sdkResource,
+                testBuildUrl: element.buildUrl,
+                compare: element.aggregateInfo[0].metrics[metric].value.mean
+            };
+        } else {
+            return {baselineScore: element.aggregateInfo[0].metrics[metric].value.mean, 
+                baselineJdkDate: element.aggregateInfo[0].benchmarkProduct.split("-")[1],
+                baselineJdk: element.aggregateInfo[0].benchmarkProduct,
+                baselineCI: element.aggregateInfo[0].metrics[metric].value.CI,
+                baselineSdkResource: element.sdkResource,
+                baselineBuildUrl: element.buildUrl,
+                compare: element.aggregateInfo[0].metrics[metric].value.mean
+            };
+
+        }
+    }
+
+    populateTable(data, type) {
+    	/* Entry format, each entry in the array is an object with two fields benchmarkName and platforms
+    	platforms is an object with each field being a separate platform containing the jdk data such as score, date, CI */
+        const newArray = [];
+        let entry = {};
```

Comment:
Maybe use `newObj` instead of `entry`. We should change all other functions as well for this.